src/core/shiftManagement.ts

```diff
@@ -9,14 +9,21 @@ import { prisma } from "../prismaClient";
 
 type Timestamps = "createdAt" | "updatedAt";
 
+type GenericSuccessResponse = {
+  status: "ok";
+};
+
 export function listAllShifts(): Promise<Shift[]> {
   return prisma.shift.findMany();
 }
 
 export function listHealthCareFacilityShifts(
   facilityUuid: HealthCareFacility["uuid"],
 ): Promise<Shift[]> {
-  return prisma.shift.findMany({ where: { facilityUuid } });
+  return prisma.shift.findMany({
+    where: { facilityUuid },
+    include: { shiftAssignments: true },
```

Comment:
Why is this additional fetch needed ?

Reply:
🔴

Reply:
Yes, this is the part of that breaking change. I did note this.